eventkit_cloud/ui/static/ui/app/tests/StatusDownloadPage/DataPackDetails.spec.tsx

```diff
@@ -133,37 +128,6 @@ describe('DataPackDetails component', () => {
         expect(instance.getTableCellWidth()).toEqual('120px');
     });
 
-    it('isZipFileCompleted should return true or false', () => {
```

Comment:
Tests relating to download button moved to their own test file as the button has been moved to a separate component..